### PR TITLE
Add Layer-Filter Extension READMEs

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: first-party/jib-layer-filter-extension-gradle/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Jib Layer-Filter Extension
+
+A general-purpose layer-filter extension that enables fine-grained layer control, including deleting files and moving files into new layers. Note that the extension can only filters files put by Jib; it cannot filter files in the base image.
+
+## Examples
+
+```grdle
```
Comment (loosebazooka) [inline first-party/jib-layer-filter-extension-gradle/README.md:7]: `grdle` -> `gradle`

also should we include how to include the actual jar into the build?

File: first-party/jib-layer-filter-extension-maven/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Jib Layer-Filter Extension
+
+A general-purpose layer-filter extension that enables fine-grained layer control, including deleting files and moving files into new layers. Note that the extension can only filters files put by Jib; it cannot filter files in the base image.
+
+## Examples
+
+```xml
```
Comment (loosebazooka) [inline first-party/jib-layer-filter-extension-maven/README.md:7]: similar to gradle question, should we add in the `<dependencies>` block

File: first-party/jib-layer-filter-extension-gradle/README.md (hunk 1 above)
Comment (chanseokoh) [reply first-party/jib-layer-filter-extension-gradle/README.md]: That's a good idea. Also updated the Ownership extension docs.

File: first-party/jib-layer-filter-extension-maven/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<pluginExtension>
+  <implementation>com.google.cloud.tools.jib.maven.extension.layerfilter.JibLayerFilterExtension</implementation>
+  <configuration implementation="com.google.cloud.tools.jib.maven.extension.layerfilter.Configuration">
+    <filters>
+      <!-- Delete all jar files. -->
+      <filter>
+        <glob>**/*.jar</glob>
+      </filter>
+      // However, retain and move google-*.jar into the new "google libraries" layer.
```
Comment (loosebazooka) [inline first-party/jib-layer-filter-extension-maven/README.md]: use xml comment style?

File: first-party/jib-layer-filter-extension-maven/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+A general-purpose layer-filter extension that enables fine-grained layer control, including deleting files and moving files into new layers. Note that the extension can only filters files put by Jib; it cannot filter files in the base image.
+
+## Examples
+
+```xml
+<pluginExtension>
+  <implementation>com.google.cloud.tools.jib.maven.extension.layerfilter.JibLayerFilterExtension</implementation>
+  <configuration implementation="com.google.cloud.tools.jib.maven.extension.layerfilter.Configuration">
+    <filters>
+      <!-- Delete all jar files. -->
```
Comment (loosebazooka) [inline first-party/jib-layer-filter-extension-maven/README.md]: This confuses me, especially given that we say: 

> If multiple filters match a file, the last filter in the order applies to the file.

does this end up being a no-op is all jars are matched below?

File: first-party/jib-layer-filter-extension-maven/README.md (hunk 3 above)
Comment (chanseokoh) [reply first-party/jib-layer-filter-extension-maven/README.md]: facepalm

File: first-party/jib-layer-filter-extension-maven/README.md (hunk 4 above)
Comment (chanseokoh) [reply first-party/jib-layer-filter-extension-maven/README.md]: Updated to "Delete all jar files (unless they match the filters below)."

So, only deleted will be the JARs not matching the filters below.

**Pull request merged**